accelerator/abstract_accelerator.py

```diff
@@ -12,7 +12,12 @@ class DeepSpeedAccelerator(ABC):
     def __init__(self):
         self._name = None
         self._communication_backend_name = None
-
+        self._capabilities: dict[str, bool] = {
+            "zero1": False,
```

Comment:
These hardcoded constants, such as "zero1", should be defined as symbolic constants in a global location.

Reply:
@tjruwase  As we discussed, I collected the names in *OpBuilder and put it in constants.py. Do we need add other features like optimizers as ds features too?